internal/pkg/cli/recorder/bpf/recorder.bpf.c

```diff
@@ -0,0 +1,47 @@
+#include <vmlinux.h>
+
+#include <asm-generic/errno.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/bpf_helpers.h>
+
+#define MAX_ENTRIES 8 * 1024
+#define MAX_SYSCALLS 1024
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
```

Comment:
Is this standard for BPF files? thought we'd be using Apache v2.

Reply:
We can't use Apache here. The kernel part has to be GPL, so we can use plain GPL, MIT/GPL or BSD/GPL